lib/foodcritic.rb

```diff
@@ -5,17 +5,4 @@
 require 'yajl'
 require 'erubis'
 
-require_relative 'foodcritic/chef'
-require_relative 'foodcritic/command_line'
-require_relative 'foodcritic/domain'
-require_relative 'foodcritic/error_checker'
-require_relative 'foodcritic/notifications'
-require_relative 'foodcritic/ast'
-require_relative 'foodcritic/xml'
-require_relative 'foodcritic/api'
-require_relative 'foodcritic/dsl'
-require_relative 'foodcritic/linter'
-require_relative 'foodcritic/output'
-require_relative 'foodcritic/rake_task'
-require_relative 'foodcritic/template'
-require_relative 'foodcritic/version'
+require_relative 'foodcritic'
```

Comment:
Ignore me

Sent from my iPhone

> On Jun 29, 2016, at 3:49 PM, Lamont Granquist <notifications@github.com> wrote:
> 
> In lib/foodcritic.rb:
> 
> > @@ -5,17 +5,4 @@
> >  require 'yajl'
> >  require 'erubis'
> >  
> > -require_relative 'foodcritic/chef'
> > -require_relative 'foodcritic/command_line'
> > -require_relative 'foodcritic/domain'
> > -require_relative 'foodcritic/error_checker'
> > -require_relative 'foodcritic/notifications'
> > -require_relative 'foodcritic/ast'
> > -require_relative 'foodcritic/xml'
> > -require_relative 'foodcritic/api'
> > -require_relative 'foodcritic/dsl'
> > -require_relative 'foodcritic/linter'
> > -require_relative 'foodcritic/output'
> > -require_relative 'foodcritic/rake_task'
> > -require_relative 'foodcritic/template'
> > -require_relative 'foodcritic/version'
> > +require_relative 'foodcritic'
> huh?
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub, or mute the thread.
>